nx-dev/ui-powerpack/src/lib/powerpack-pricing.tsx

```diff
@@ -12,7 +12,7 @@ export function PowerpackPricing() {
       <div className="flex flex-col gap-2">
         <div className="group relative flex w-full items-center justify-between gap-8 rounded-md border border-slate-200 bg-white px-6 py-4 transition hover:bg-white/90 hover:shadow dark:border-slate-800/60 dark:bg-slate-900/60 dark:hover:bg-slate-900/100">
           <a
-            href="https://cloud.nx.app/powerpack/purchase?utm_source=nx.dev&utm_medium=referral&utm_campaign=nx-powerpackurl"
+            href="https://cloud.nx.app/powerpack/purchase?utm_source=nx-website&utm_medium=referral&utm_campaign=powerpack-landing-page&utm_content=cta-button&utm_term=purchase-powerpack-annually"
```

Comment:
if we switch `nx.dev` => `nx-website` we should replace it across the repo I feel.

Reply:
on the homepage we also use `homepage` as the source which should be `nx-website` now I guess